### ISSUE TITLE
Update package with new version of flutter map

### ISSUE THREAD

**Issue by ParhaMDeF** [ankiimation/flutter_map_animated_marker#1]
hi sir.
I need your package but this is not supported by flutter map version 3.0.0.
can you solve it?
thanks.

```
Because flutter_map_animated_marker >=0.0.4 depends on flutter_map ^2.2.0 and wefinder depends on flutter_map ^3.0.0, flutter_map_animated_marker >=0.0.4 is
  forbidden.
So, because wefinder depends on flutter_map_animated_marker ^0.0.4, version solving failed.
```

**Comment by ankiimation** [comment]
thanks, i will check it

**Comment by ankiimation** [comment]
version 1.0.0 published with flutter_map 3.0.0 supported

**Comment by ParhaMDeF** [comment]
thanks a lot

**Comment by ParhaMDeF** [comment]
> version 1.0.0 published with flutter_map 3.0.0 supported

sorry but I think you need to update the documentation of the package.

there is not any plugin type in `MapOption`

**Comment by ankiimation** [comment]
> > version 1.0.0 published with flutter_map 3.0.0 supported
> 
> sorry but I think you need to update the documentation of the package.
> 
> there is not any plugin type in `MapOption`

thanks, document updated

**Comment by ParhaMDeF** [comment]
> > > version 1.0.0 published with flutter_map 3.0.0 supported
> > 
> > 
> > sorry but I think you need to update the documentation of the package.
> > there is not any plugin type in `MapOption`
> 
> thanks, document updated

Thank you so much. you are amazing!🚀
